pkg/blackduck/ctl_blackduck_test.go

```diff
@@ -379,16 +368,6 @@ func TestSetFlag(t *testing.T) {
 			changedSpec: &blackduckv1.BlackduckSpec{PersistentStorage: true},
 		},
 		// case
```

Comment:
not for this PR, but just for future testing, we can add example test data files for these flags.  I can work on this.